ee/tabby-webserver/src/avatar.rs

```diff
@@ -0,0 +1,30 @@
+use std::sync::Arc;
+
+use axum::{
+    extract::{Path, State},
+    response::Response,
+};
+use hyper::{header::CONTENT_TYPE, Body, StatusCode};
+use juniper::ID;
+use tracing::error;
+
+use crate::schema::auth::AuthenticationService;
+
+pub async fn avatar(
```

Comment:
add unit test coverage

Reply:
Please review commit `cadd57669020f429bfb0f568b4271500d84fcbef`, I had to change a number of things in order to get a test implemented

Reply:
that's fine. will refine ast-grep rule a bit.